src/wopiserver.py

```diff
@@ -108,6 +108,9 @@ def init(cls):
                 + hostname + '", "level": "%(levelname)s", "process": "%(name)s", %(message)s}',
                 datefmt='%Y-%m-%dT%H:%M:%S'))
             cls.app.logger.handlers = [loghandler]
+            if cls.config.get('general', 'internalserver', fallback='flask') == 'waitress':
+                cls.log.logger.handlers.clear()
+                logging.getLogger().handlers = [loghandler]
```

Comment:
Just for polishing the code, why not dropping line 110 and do:
```
if <waitress>:
    logging.getLogger().handlers = [loghandler]
else:
    cls.app.logger.handlers = [loghandler]
```

Reply:
That doesn't work as flask has automatically registered a logger at `app.logger`.

Reply:
I see, so the code should actually be:
```
if <waitress>:
    cls.log.logger.handlers.clear()
    logging.getLogger().handlers = [loghandler]
else:
    cls.app.logger.handlers = [loghandler]
```
My point is to make it straight out of the code that `cls.app.logger.handlers = [loghandler]` applies to Flask only.

Reply:
Ok, changed.